infrastructure/zk/src/format_sql.ts

```diff
@@ -0,0 +1,147 @@
+import * as fs from 'fs';
+import * as utils from './utils';
+import { format } from 'sql-formatter';
+
+function formatQuery(query: string) {
+    let formattedQuery = query;
+    try {
+        formattedQuery = format(query, {
+            language: 'postgresql',
+            tabWidth: 4,
+            keywordCase: 'upper',
+            expressionWidth: 80,
+            indentStyle: 'tabularRight'
```

Comment:
Bikeshedding: IMO, the standard ident style is more readable; `tabularRight` produces fewer lines, but because of right-aligning some keywords, it has some unexpected (personal opinion) spacing, such as one at the start of the first line.

Reply:
As this is something that is very personal, I will make an experiment and let the team vote on slack to choose the formatting style